sources/iTermColorMap.m

```diff
@@ -113,7 +125,7 @@ - (NSColor *)colorForKey:(iTermColorMapKey)theKey {
         return [NSColor colorWith8BitRed:red green:green blue:blue];
     } else {
         NSColor *result = _map[@(theKey)];
-        if (!result) {
+        if (!result && theKey != kColorMapUnderline) {
```

Comment:
If I return result directly, I get a crash right after startup that seems unrelated to the changes in this branch:

```
        0   CoreFoundation                      0x00007fff971374f2 __exceptionPreprocess + 178
        1   libobjc.A.dylib                     0x00007fff9666773c objc_exception_throw + 48
        2   CoreFoundation                      0x00007fff9719e4bd +[NSException raise:format:] + 205
        3   AppKit                              0x00007fff8e7603e0 -[NSColorSpaceColor initWithColorSpace:components:count:] + 120
        4   AppKit                              0x00007fff8e760295 +[NSColor colorWithColorSpace:components:count:] + 29
        5   iTerm2                              0x000000010ecfcb69 -[iTermColorMap processedBackgroundColorForBackgroundColor:] + 953
        6   iTerm2                              0x000000010eba766b -[PTYTextView defaultBackgroundColor] + 203
        7   iTerm2                              0x000000010eba78e2 -[PTYTextView updateMarkedTextAttributes] + 66
        8   iTerm2                              0x000000010eba636b -[PTYTextView initWithFrame:colorMap:] + 395
        9   iTerm2                              0x000000010eb6d6c7 -[PTYSession setScreenSize:parent:] + 1143
        10  iTerm2                              0x000000010ece5068 -[PseudoTerminal setupSession:title:withSize:] + 2216
        11  iTerm2                              0x000000010eceb9cf -[PseudoTerminal addSessionInNewTab:] + 191
        12  iTerm2                              0x000000010ecea964 -[PseudoTerminal createTabWithProfile:withCommand:] + 1236
        13  iTerm2                              0x000000010ec4bedd -[iTermController launchBookmark:inTerminal:withURL:isHotkey:makeKey:canActivate:command:block:] + 1229
        14  iTerm2                              0x000000010ec4b4a7 -[iTermController launchBookmark:inTerminal:] + 119
        15  iTerm2                              0x000000010ec45f66 -[iTermController newWindow:possiblyTmux:] + 182
        16  iTerm2                              0x000000010ed05e08 -[iTermApplicationDelegate newWindow:] + 104
        17  iTerm2                              0x000000010ed01ec9 -[iTermApplicationDelegate performStartupActivities] + 873
```

Any idea?